packages/bundles/kairos-overlay-files/files/system/oem/11_persistent.yaml

```diff
@@ -11,3 +11,6 @@ stages:
         expand_partition:
```

Comment:
expand_partition works only on GPT, if we call growpart it would work as well for RPI3

Reply:
?

This is the yip plugin which works only on GPT. For MBR there is nothing that can expand or create partitions currently.

Reply:
nope but just by using growpart+resize2fs would do the trick for both MBR and GPT